device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/mqtt/MqttDeviceTwin.java

```diff
@@ -405,6 +405,7 @@ else if (topic.length() > PATCH.length() && topic.startsWith(PATCH))
         }
     }
 
+    @SuppressWarnings("SameParameterValue") // This method can create a new TransportException with any message string.
```

Comment:
what's the warning here?

Reply:
the warning here specifies that we currently have a single usage for this method, and thus, a single value for "message". As with all other generic methods, "message" can be anything.

Reply:
Nit: add - that we currently have a single usage for this method